src/common/JSHandle.ts

```diff
@@ -791,15 +791,12 @@ export class ElementHandle<
    * the return value resolves to `[]`.
    */
   async $$(selector: string): Promise<ElementHandle[]> {
-    const defaultHandler = (element: Element, selector: string) =>
-      element.querySelectorAll(selector);
     const { updatedSelector, queryHandler } = getQueryHandlerAndSelector(
-      selector,
-      defaultHandler
+      selector
     );
 
     const arrayHandle = await this.evaluateHandle(
```

Comment:
Just a quick suggestion: I know the variable name `arrayHandle` is from the old code, but we can consider changing it to something like `handles` or `handleList`, to be more standard IMO.

Reply:
fixed